udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -362,7 +377,9 @@ if(!dataset.spatial) {
     zones: undefined,
     granularity: undefined,
   }
-}
+};
+
+const userOrganization = ref<Organization>();
```

Comment:
It seems that `userOrganization` is the `id` and not the entire object from the multiselect so this one should be a `ref<string>`